doozer/doozerlib/olm/bundle.py

```diff
@@ -29,77 +29,57 @@ class OLMBundle(object):
     independently built, due to their tight coupling to corresponding operators
     """
 
-    def __init__(self, runtime: Runtime, dry_run: bool, brew_session: Optional[ClientSession] = None):
+    def __init__(self, runtime: Runtime,
+                 operator_nvr_or_dict: Union[str, dict],
```

Comment:
Make the object specific to an nvr/build and init with it. This cleans the whole object and its methods and we are not setting / operating on faulty assumptions.